Makefile

```diff
@@ -1,4 +1,4 @@
-export BOTNAME := limbo-travisci
+export BOTNAME ?= limbo-travisci
```

Comment:
I can't figure out why this shows as a delta from the master, because the master also has this change.